manifests/install.pp

```diff
@@ -83,10 +82,15 @@
     group  => $crowd::group,
   }
 
+  # symlink all logfiles to logging directory
```

Comment:
This doesn't fit in this PR (unrelated). Some work's been done in recent commits to customize the logging directory.